website/server/controllers/api-v3/auth.js

```diff
@@ -130,6 +130,14 @@ api.registerLocal = {
       newUser.registeredThrough = req.headers['x-client']; // Not saved, used to create the correct tasks based on the device used
     }
 
+    // A/B Test 2016-09-12: Start with Sound Enabled?
+    if (Math.random() < 0.5) {
+      newUser.preferences.sound = 'rosstavoTheme';
+      newUser._ABtest = '20160912-soundEnabled';
+    } else {
+      newUser._ABtest = '20160912-soundDisabled';
+    }
+
```

Comment:
Suggestions:
- Move the ABtest logic out of here.
- Have some sort of collection of possible ABtests (right now, it would only have this one), where each test has a set of possible options.
- Choose one test at random, then choose one option at random.

That way, this logic would allow for multiple ABtests at once, and the logic would be modular, hidden away elsewhere.